charts/global-network-policies/templates/allow-to-aggregate-prometheus.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: {{ include "networkpolicyversion" . }}
+kind: NetworkPolicy
+metadata:
+  annotations:
+    gardener.cloud/description: |
+      Allows Egress from pods labeled with 'networking.gardener.cloud/to-aggregate-prometheus=allowed'
+      to all the shoot apiservers running in the seed cluster.
```

Comment:
"to all the shoot apiservers running in the seed cluster." ?

Reply:
My bad. Didn't change the description. I will fix it.